base/float.jl

```diff
@@ -113,6 +113,11 @@ for T in (Float16, Float32, Float64)
     @eval exponent_raw_max(::Type{$T}) = $(Int(exponent_mask(T) >> significand_bits(T)))
 end
 
+# The minimum exponent for a normal number. Assuming unknown types
+# don't have subnormals.
+min_normal_number_exponent(::Type{<:AbstractFloat}) = nothing
+min_normal_number_exponent(::Type{F}) where {F<:IEEEFloat} = true - exponent_bias(F)
```

Comment:
```suggestion
min_normal_number_exponent(::Type{F}) where {F<:IEEEFloat} = 1 - exponent_bias(F)
```

This always returns an `Int` anyway.